enchant.lua

```diff
@@ -574,34 +575,35 @@ end
 --  @return False with an error message if a probem occurs, 
 --  @return true with the name of the item enchanted and the enchantment name if successful
 --  
-local function EnchantItem( Split )
+local function EnchantItem( PlayerName, EnchantmentIDString, LevelString )
 
-	local PlayerName = Split[2]
-	local lcPlayerName = string.lower(PlayerName)
-	local EnchantmentID = cEnchantments:StringToEnchantmentID( Split[3] )
+	local EnchantmentID = cEnchantments:StringToEnchantmentID( EnchantmentIDString )
 	local EnchantmentName
-	local Level = tonumber( Split[4] or 1 )
+	local ItemName
+	local Level = tonumber( LevelString )
+	local IsSuccess = false
 
 	if not Level then
-		return false, string.format( LevelNAN, Split[4] )
-	elseif Level == 0 then
-		return false, LevelIsZero
+		return string.format( LevelNAN, LevelString )
```

Comment:
The doxy-comment for the entire function is now wrong about the returned values. Please fix that altogether.

Reply:
Good catch, I missed that. I haven't set up or used Doxygen, so please let me know if the syntax is incorrect.